src/SDKs/MixedReality/Management.MixedReality/Management.MixedReality.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.MixedReality</PackageId>
+    <Description>Microsoft Azure Management Mixed Reality Library</Description>
+    <Version>0.9.0-preview</Version>
+    <AssemblyName>Microsoft.Azure.Management.MixedReality</AssemblyName>
+    <RootNamespace>Microsoft.Azure.Management.MixedReality</RootNamespace>
+    <PackageTags>MixedReality</PackageTags>
+    <PackageReleaseNotes>
+<![CDATA[
+  This is the client library of Mixed Reality Azure Resource Management API.
+  
+  Operations could be performed by using this client library:
+  
+  1. List all available operations;
+  2. Create/Delete specified Spatial Anchors Account;
+  3. Add/Modify/Remove tags of specified Spatial Anchors Account;
+  4. List all Spatial Anchors Account(s) in certain Resource Group or Subscription.
+]]>
+    </PackageReleaseNotes>
+  </PropertyGroup>
+
+  <PropertyGroup>
```

Comment:
@XiangyuL-Microsoft take a look at other projects and model your project file accordingly.
You cannot specify target fx in your project file, plus now all .NET SDKs support more target fx than just these two.
Look at https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj
and model your project file accordingly.